src/main/java/tools/jackson/databind/cfg/MapperBuilder.java

```diff
@@ -241,6 +244,12 @@ public abstract class MapperBuilder<M extends ObjectMapper,
      */
     protected int _formatWriteFeatures;
 
+    /**
+     * States of {@link JavaTimeFeature}s to enable/disable.
+     */
+    protected JacksonFeatureSet<JavaTimeFeature> _javaTimeFeatures =
```

Comment:
Ahhhh. Ok, here's something we'll need to change: new `JavaTimeFeature` (or eventual `DateTimeFeature`) can and should be added as 3rd thing in `DatatypeFeatures` (along with `EnumFeature` and `JsonNodeFeature`) --  it's designed to be extended.

I can do that tomorrow.